src/App.tsx

```diff
@@ -15,71 +17,99 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
```

Comment:
Replace with `goods[goods.indexOf('Jam')] || goods[0]`. This is a safer case because if there is such an element in the array, it will choose the first one.